spotty/spotty/UIComponents/Styling/Enums/LabelStyle.swift

```diff
@@ -29,12 +31,14 @@ enum LabelStyle {
             return .preferredFont(forTextStyle: .subheadline)
         case .largeCellTitle, .largeCellSubtitle:
             return .preferredFont(forTextStyle: .title2)
+        case .detailViewTitle, .detailItemTitle:
+            return UIFontMetrics.default.scaledFont(for: UIFont.systemFont(ofSize: 35, weight: .bold))
         }
     }
     
     var textColor: UIColor {
         switch self {
-        case .sectionHeaderTitle, .mediumCellTitle, .largeCellTitle, .playlistDescription:
+        case .sectionHeaderTitle, .mediumCellTitle, .largeCellTitle, .playlistDescription, .detailViewTitle, .detailItemTitle:
```

Comment:
We suggest having up to 4 cases in a single line.

Reply:
done